x-pack/plugins/triggers_actions_ui/public/application/sections/alert_form/alert_add.tsx

```diff
@@ -122,8 +117,16 @@ export const AlertAdd = ({
       }
       return newAlert;
     } catch (errorRes) {
-      setServerError(errorRes);
-      return undefined;
+      if (toastNotifications) {
```

Comment:
Looks like you don't need the `if` check on `toastNotifications` here - or I missed something.  There are other references to `toastNotifications` that aren't guarded, and from what I can see in my current branch, it's not an optional type.  And there's other guarded usage in this PR ...

But I'm curious - if it's possible for this to be false, then I guess the error is silently dropped?  Another good reason to remove the `if` check :-)

Reply:
I just checked, looks like `toastNotifications` recently became mandatory here: https://github.com/elastic/kibana/commit/5ff13ada6b3b8a48d9bf9f8fca79421cedbc7940#diff-982166978fb5c047366daa491e782771R19.

Since my code was older, it was doing the `if` check but now I will change it 🙂

Reply:
Fixed in https://github.com/elastic/kibana/pull/59444/commits/193d50fe1a48345cef05c488628f89eba1090474